src/base/path.cpp

```diff
@@ -218,6 +218,15 @@ QString Path::toString() const
     return QDir::toNativeSeparators(m_pathStr);
 }
 
+std::filesystem::path Path::toStdFsPath() const
+{
+#ifdef Q_OS_WIN
+    return data().toStdWString();
```

Comment:
Is std path OK with forward slashes on Windows?

Reply:
Yes and it seems I should properly pass this information to its constructor. Will push update later.

Reply:
Or maybe `return toString().toStrWString();`?